### ISSUE TITLE
Asus Zenbook UX433FN support (Whiskey Lake)

### ISSUE THREAD

**Issue by TheNewJavaman** [hieplpvip/ASUS-ZENBOOK-UX410-HACKINTOSH#92]
**What is your laptop?**
Asus Zenbook UX433FN

**Full specs**
i7-8565U (Whiskey Lake, 4 cores)
Intel UH 620 + MX150 (but I disable egpu in boot flags for macOS)
1920x1080 standard display
512GB NVME PCIe SSD
Intel WiFi, but it works with [AirportItlwm.kext](https://github.com/OpenIntelWireless/itlwm)
[Full spec list on Asus's website](https://www.asus.com/us/Laptops/ASUS-ZenBook-14-UX433FN/Tech-Specs/)

**Attach original DSDT from Clover**
Never used Clover, only using OpenCore

**Attach full report from AIDA64 (if possible)**
Not possible, sorry

**Additional comments**
I am almost able to get a fully working EFI set for my computer not using this repo. However, keyboard functions are mapped improperly or nonexistent. What part of this repo's `config.plist` adds in proper keyboard support?

**Comment by andreademasi** [comment]
Hi @TheNewJavaman , did you manage to get the keyboard to work properly?

**Comment by TheNewJavaman** [comment]
@andreademasi Yes! Take a look at [my EFI config](https://github.com/TheNewJavaman/asus-zenbook-ux433fn-hackintosh-opencore-efi) for a complete example.
